Backend/app/spotify_electron/user/user/user_service.py

```diff
@@ -187,3 +188,39 @@ def search_by_name(name: str) -> list[UserDTO]:
             f"Unexpected error in User Service getting items by name {name}"
         )
         raise UserServiceException from exception
+
+
+# TODO make it a transaction
+def upgrade_user_to_artist(user_name: str) -> None:
+    """Upgrade user account to artist account
+
+    Args:
+        user_name (str): user name
+
+    Raises:
+        UserNotFoundException: if the user does not exist
+        UserServiceException: unexpected error while upgrading user to artist
+    """
+    try:
+        base_user_service.validate_user_name_parameter(user_name)
```

Comment:
After this we should add a check for user not existing on the database. The function already exists.

This Will introduce a new exception that has to be documented in controller and service methods